activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   Introduce ApplicationRecord, an Active Record layer supertype
+
+    It's pretty common for folks to monkey patch `ActiveRecord::Base` to
+    work around an issue or introduce extra functionality. Instead of
+    shoving even more stuff in `ActiveRecord::Base`, `ApplicationRecord` can
+    hold all those custom work the apps may need.
```

Comment:
Yes! We can and should advertise that to plugin authors. I have AR extensions myself, which I can distribute as modules to be mixed into `ApplicationRecord` instead of `ActiveRecord::Base` monkey patches.